pkg/controller/windowsmachine/windows/windows.go

```diff
@@ -227,6 +233,23 @@ func (vm *windows) ConfigureHybridOverlay(nodeName string) error {
 	return nil
 }
 
+func (vm *windows) ConfigureWindowsExporter() error {
+	// configures service to collect and expose metrics at endpoint with default port :9182 and default URL path /metrics
+	windowsExporterServiceArgs := "--collectors.enabled " +
+		"cpu,cs,logical_disk,net,os,service,system,textfile,container,memory"
+
+	windowsExporterService, err := newService(WindowsExporterPath, windowsExporterServiceName, windowsExporterServiceArgs)
```

Comment:
Does this service depend on any other service to start before it can work? Example: can this service run before the kubelet service is running?